Automattic/wp-calypso
# Social Connections: Fix Instagram issues

This change fixes up some issue we have on the connections screen

## Proposed Changes

* Remove video reels from description as its not developed yet
* Adds the new logo to Mastodon
* Adds a default profile picture for Instagram and Mastodon if there is none present.

## Testing Instructions



* Open up calypso live for your site, which has the Mastodon blog sticker and newest version of Jetpack.
* Check the connections screen, it should look like this:


![CleanShot 2023-06-08 at 08 08 22 png](https://github.com/Automattic/wp-calypso/assets/36671565/583f17a8-951e-4fc7-b258-b28dae7ded35)

## Pre-merge Checklist
